sql_script/create_tables.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS user (
+    user_id VARCHAR(64) PRIMARY KEY,
```

Comment:
userID+userType are unique key
i am not sure if there is a userID collision between two userType